### ISSUE TITLE
Unwrap panic

### ISSUE THREAD

**Issue by sjud** [tjardoo/openai-client#40]
```
thread '<unnamed>' panicked at /Users/sjud/.cargo/registry/src/index.crates.io-6f17d22bba15001f/openai_dive-0.3.5/src/v1/resources/shared.rs:109:18:
```

for reference

```rust
 x_ratelimit_limit_requests: value
                .get("x-ratelimit-limit-requests")
                .unwrap()
```

I can make a pull request for error handling here if helpful?

**Comment by tjardoo** [comment]
Thanks! That'd be helpful. It fails on `src\v1\api.rs:107` because not all endpoints return the headers.

**Comment by tjardoo** [comment]
This issue should now be fixed with #41
